docs/decisions/0018-standarize-django-po-files.rst

```diff
@@ -0,0 +1,152 @@
+  translation merger step altogether.
+- Translators will need one workflow for both latest (master) and named
+  releases for the ``edx-platform`` repo.
+- Simplifies the work of both the Build Test Release and Transifex working groups.
+
+Cons:
+
+- It will be harder for translators to focus on learner-facing text and de-prioritize
+  educator-facing text.
+
+
+Rejected Alternatives
+---------------------
+
+Combine into four files
+^^^^^^^^^^^^^^^^^^^^^^^
+
+This option tries to keep the original idea of splitting to allow translators
+to focus on learner-facing text and de-prioritize educator-facing text.
+
+The four files will be ``platform.po``, ``platform-js.po``, ``studio.po``
+and ``studio-js.po``.
+
+It will not concern the translators with the technical split
+of the files into: ``wiki.po``, ``mako.po``, and ``django-partial.po``, etc.
+
+
+In this option we'll create two new configuration files
+``config.extract-oep58.yaml`` and ``config.pull-oep58.yaml`` to combine the
+pofiles into the four files and then combine them into two files respectively:
+
+.. code:: yaml
+
+    # config.extract-oep58.yaml
+    # This file is used by the ``make extract_translations`` when
+    # the OPENEDX_COMBINE_FILES environment variable is enabled.
+    generate_merge:
+        platform.po:
+            - django-partial.po
+            - mako.po
+            - wiki.po
+            - edx_proctoring_proctortrack.po
+        platform-js.po:
+            - djangojs-partial.po
+            - djangojs-account-settings-view.po
+            - underscore.po
+        studio.po:
+            - django-studio.po
+            - mako-studio.po
+        studio-js.po:
+            - djangojs-studio.po
+            - underscore-studio.po
+
+A corresponding ``Makefile`` change is needed:
+
+.. code:: make
+
+    extract_translations: ## extract localizable strings from sources
+        i18n_tool extract -v
+        if [ -z "$$OPENEDX_COMBINE_FILES" ]; then \
+            i18n_tool generate --config=config.extract-oep58.yaml --verbose 1;
+        fi
+
+The other file would be ran after ``make pull_translations`` which will
+be used to make the final ``django.po`` and ``djangojs.po`` files that are
+usable by Django:
+
+.. code:: yaml
+
+    # config.pull-oep58.yaml
+    # This file is used by the ``make pull_translations`` when
+    # the OPENEDX_ATLAS_PULL environment variable is enabled.
+    generate_merge:
+        django.po:
+            - platform.po
+            - studio.po
+        djangojs.po:
+            - platform-js.po
+            - studio-js.po
+
+
+    pull_translations: ## extract localizable strings from sources
+        if [ -z "$$OPENEDX_ATLAS_PULL" ]; then \
+            atlas pull translations/edx-platform/conf/locale
+            i18n_tool --config=config.pull-oep58.yaml generate --verbose 1;
+        fi
+
+
+This option adds more complexity, but retain the Translator-friendly resource
+split.
```

Comment:
After reading this discussion, I think this rejected alternative section could also have an explanation for why it was rejected (still too much complexity, wanting to simplify as much as possible, etc).

Reply:
Thanks @dianakhuang. I'll expand this paragraph.

Reply:
Done @dianakhuang, would you mind taking another look?